bbc/simorgh
# Extend tap targets for Footer links

The tap target of list items in the footer now span the width of the entire list item.

* [x] Fixes https://github.com/bbc/simorgh/issues/393


**Test Instructions**
1. Git checkout tap-targets-footer-links.
2. `npm run storybook`.
3. Navigate onto the footer story.
4. Touch targets should match the UX specification.
